cogs/squid.py

```diff
@@ -0,0 +1,38 @@
+# imports to make the dream
+
+import random
+import asyncio
+import discord
+import commands
+
+''' dab '''
+class dab:
+    def __init__(self, bot):
+        self.bot = bot
+    
+    @commands.command(name = "squid", pass_context = True)
+    async def mycom(self):
```

Comment:
If you have `pass_context=True`, then you need an additional argument, which is usually named `ctx`.  Alternatively, remove the `pass_context=True` in the decorator since you're not using any context in your method below, and keep this declaration the same.